package.json

```diff
@@ -13,13 +13,13 @@
     "url": "https://github.com/GPII/gpii-mail-test"
   },
   "dependencies": {
-    "infusion": "2.0.0-dev.20151211T025229Z.eabb4fc",
+    "infusion": "git://github.com/amb26/infusion.git#81ef49cdc0e78a96549c5195d9ffd42604087096",
```

Comment:
Thanks.  I included the update in an unrelated PR for the package, it works fine.  Will push it through along with the versioned reference to gpii-express.